src/utils/useTxMulti/useTxMulti.spec.tsx

```diff
@@ -0,0 +1,218 @@
+import React from 'react';
+import { renderHook, act } from '@testing-library/react-hooks';
+import isEmpty from 'ramda/src/isEmpty';
+import { waitFor } from 'test-utils';
+
+import { fAccount, fAccounts, fAssets, fNetwork, fNetworks } from '@fixtures';
+import { ITxObject, ITxToAddress, ITxValue, ITxData, ITxHash, ITxType, ITxStatus } from '@types';
+import { AccountContext, DataContext, IDataContext, StoreContext } from '@services';
+
+import { useTxMulti } from './useTxMulti';
+
+const createTxRaw = (idx: number): Partial<ITxObject> => ({
+  to: ('address' + idx) as ITxToAddress,
```

Comment:
Why `ITxToAddress` instead of `TAddress`?

Reply:
That's the type on `ITxObject`